internal/binutils/binutils.go

```diff
@@ -179,6 +177,25 @@ func findObjdump(paths []string) (string, bool, bool) {
 	return "", false, false
 }
 
+// chooseExe finds and returns path to preferred binary. names is a list of
+// names to search on both Linux and OSX. osxNames is a list of names specific
+// to OSX. names always has a higher priority than osxNames. The order of
+// the name within each list decides its priority (e.g. the first name has a
+// higher priority than the second name in the list).
+// It returns a string with path to the binary if found, or an empty string if
```

Comment:
I would skip explaining what is returned when no binary is found - it's pretty self-explanatory IMO that the first return value is undefined if "false" is returned in the second value.

Reply:
Done.